### ISSUE TITLE
The manager user created through users.xml does not have full permissions

### ISSUE THREAD

**Issue by zxl-wangwang** [ClickHouse/ClickHouse#70999]
### Question

clickhouse version 22.8:

The manager user is created through the configuration file, and access_management is 1.
At this point, the manager user has all permissions.

![截屏2024-10-24 10 02 17](https://github.com/user-attachments/assets/7b882cd8-eaf0-4cc6-a2c7-87aabfb7ce38)

clickhouse version 23.8:

The manager user is created through the configuration file, and access_management is 1.
However, the manager user does not have all permissions at this time. How can I allow the manager to have all permissions?

![截屏2024-10-24 10 03 35](https://github.com/user-attachments/assets/5818b42a-d9ba-49e6-9ac5-2641c7ed41c3)

**Comment by zxl-wangwang** [comment]
Is this problem due to version issue? What should I do if I want to obtain all permissions in version 23.8?

**Comment by zxl-wangwang** [comment]
Why do users created through configuration files in version 23.8 not have all permissions?

**Comment by den-crane** [comment]
access_management is not enough with 23.8

```
<access_management>1</access_management>
<named_collection_control>1</named_collection_control>
<show_named_collections>1</show_named_collections>
<show_named_collections_secrets>1</show_named_collections_secrets>
```

BTW 23.8 is out of support, you should upgrade to 24.8

**Comment by zxl-wangwang** [comment]
Thank you very much. What does the BTT function mean?

**Comment by den-crane** [comment]
> What does the BTT function mean?

BTW is abbreviation for "by the way". 

-----

Clickhouse server 23.8 is out of support, you should upgrade to 24.8

**Comment by zxl-wangwang** [comment]
OK, I have solved the problem in version 23.8 with the configuration you provided. Thank you very much.
